Yoast/wordpress-seo
# Move components platform specific styles

## Summary

This PR can be summarized in the following changelog entry:

Move WordPress specific styles used by components in a Sass partial.

## Relevant technical choices:

* Platform specific styles, e.g. WordPress font and links styles, should be moved out from components-specific stylesheets. This introduces a `_wp.scss` file that should be used to import WordPress styles into components that need them (e.g. the onboarding wizard).
* Worth noting this PR will also start using some styles from the new color palette merged in https://github.com/Yoast/yoast-components/pull/109

## Test instructions

This PR can be tested by following these steps:

* npm link yoast-components
* on trunk, build the CSS
* go on the onboarding wizard and observe the missing font and link styles
* switch to this branch
* go on the onboarding wizard and check styles are correct
* edit a post and check the metabox `.assessment-results__mark:focus` style is working correctly

Fixes #6046
